package.json

```diff
@@ -21,5 +21,8 @@
     "jest": "^27.1.0",
     "nodemon": "^2.0.13"
   },
-  "dependencies": {}
+  "dependencies": {
+    "axios": "^0.24.0",
+    "euclidean-distance": "^1.0.0"
```

Comment:
There are some downsides of adding packages for trivial things, can you think of some?

Reply:
I could have written a function for calculating euclidian distance myself but I checked the "euclidian-distance" npm package code and it was pretty good and the author wrote tests for it. So, instead of writing a function, I decided to use a well-tested package to be safe.

    I agree that we should not use npm packages for trivial things because they become an external dependency for the application at the end. And we have to also take care of their maintenance (version updates) and security issues.